client/go/outline/connectivity/connectivity.go

```diff
@@ -52,7 +52,10 @@ func CheckTCPAndUDPConnectivity(
 		udpErrChan <- CheckUDPConnectivityWithDNS(udp, resolverAddr)
 	}()
 
-	tcpErr = CheckTCPConnectivityWithHTTP(tcp, testTCPWebsite)
+	tcpErr = CheckTCPConnectivityWithHTTP(tcp, []string{
+		"https://connectivitycheck.gstatic.com/generate_204",
+		"https://www.google.com/generate_204",
```

Comment:
Please add `http://example.com`, so we don't introduce any regressions.

Reply:
I agree with @gh-andre , I don't think we should use `example.com`, it's not its intention and not nearly as robust as the options we have now in the PR.

I also addressed the timeout concern by racing the tests.

Reply:
My only concern with dropping `example.com` was the risk of some regions being unable to access the new URLs, even if they could reach `example.com` (like China can't access `http://www.google.com/generate_204`). However, since "geoblock.net" tests show the other three URLs are globally accessible, I'm comfortable moving forward without `example.com` for now.